db/schema.rb

```diff
@@ -161,4 +162,5 @@
   end
 
   add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
+  add_foreign_key "active_storage_variant_records", "active_storage_blobs", column: "blob_id"
```

Comment:
and thiiiiiis

Reply:
https://github.com/umts/jobapps/blob/1a8c33b31c99e6950668d840767276ee3000fdfd/db/migrate/20210224185747_create_active_storage_variant_records.active_storage.rb#L9

Reply:
Accidentally removed in #417 due to a merge error (07a9bb316497c45f90eabbc5ee0ebf754d96acaa)